tests/gis_tests/admin.py

```diff
@@ -4,3 +4,4 @@
     from django.contrib import admin
 
     admin.OSMGeoAdmin = admin.ModelAdmin
+    admin.GISModelAdmin = admin.ModelAdmin
```

Comment:
Swap it with `OSMGeoAdmin` and add a comment that will help up when deprecation ends:
```python
    admin.GISModelAdmin = admin.ModelAdmin
    # RemovedInDjango50Warning.
    admin.OSMGeoAdmin = admin.ModelAdmin
````